frontend/public/scss/receipt.scss

```diff
@@ -0,0 +1,257 @@
+
+  .receipt-col {
+    float: left;
+    width: 50%;
+    padding: 0 15px;
+
+    @media only screen and (max-width: 767px) {
+      float: none;
+      width: 100%;
+      padding-bottom: 40px;
+    }
+
+    &.p-t-50 {
+      padding-top: 50px;
+
+      @media only screen and (max-width: 767px) {
+        padding-top: 0;
+      }
+    }
+  }
+
+  h3 {
+    font: 600 20px/26px 'Rajdhani', Times New Roman, serif;
+    margin: 0 -15px 20px;
+    border: none;
+    padding: 5px 15px;
+    color: black;
+    background: lightgrey;
+
+    @media only screen and (max-width: 992px) {
+      font: 600 18px/24px 'Rajdhani', Times New Roman, serif;
+    }
+  }
+}
+
+.receipt-logo {
+  width: 150px;
+  margin: 0 0 10px;
+
+  img {
+    display: block;
+    width: 100%;
+    height: auto;
+  }
+}
+
+.receipt-mit-info {
+  max-width: 226px;
+  font-size: 16px;
+  line-height: 21px;
+
+  p {
+    margin: 0;
+  }
+}
+
+.receipt-table-holder {
+  overflow-x: auto;
+  margin: 0 -15px;
+  padding: 0 0 15px;
+
+  h3 {
+    font: 600 20px/26px 'Rajdhani', Times New Roman, serif;
+    margin: 0;
+    padding: 5px 15px;
+    color: black;
+    background: lightgrey;
+    display: none;
+    border: none;
+
+    @media only screen and (max-width: 767px) {
+      display: block;
+    }
+  }
+
+  .receipt-table {
+    width: 100%;
+    border-spacing: 0;
+
+    thead {
+
+      @media only screen and (max-width: 767px) {
+        display: none;
+      }
+    }
+
+    th {
+      padding: 5px 15px;
+      text-align: left;
+      color: black;
+      background: lightgrey;
+      white-space: nowrap;
+      font: 600 20px/26px 'Rajdhani', Times New Roman, serif;
+
+      @media only screen and (max-width: 992px) {
+        font: 600 18px/24px 'Rajdhani', Times New Roman, serif;
+      }
+    }
+
+    td {
+      padding: 20px 15px 0;
+      vertical-align: top;
+
+      @media only screen and (max-width: 767px) {
+        display: block;
+        width: 100%;
+      }
+
+      &:before {
+        display: none;
+        color: black;
+        white-space: nowrap;
+        padding: 0 10px 0 0;
+        font: 600 18px/24px 'Rajdhani', Times New Roman, serif;
+        float: left;
+        width: 135px;
+
+        @media only screen and (max-width: 767px) {
+          display: block;
+        }
+      }
+
+      div {
+        overflow: hidden;
+      }
+
+      &:nth-child(1) {
+
+        &:before {
+          content: "Title";
+        }
+      }
+
+      &:nth-child(2) {
+
+        &:before {
+          content: "Quantity";
+        }
+      }
+
+      &:nth-child(3) {
+
+        &:before {
+          content: "Unit Price";
+        }
+      }
+
+      &:nth-child(4) {
+
+        &:before {
+          content: "Discount";
+        }
+      }
+
+      &:nth-child(5) {
+
+        &:before {
+          content: "Total Paid";
+        }
+      }
+    }
+  }
+}
+
+@media print {
+  .print-btn,
+  .print-row,
+  .footer,
+  .header-holder {
+    display: none !important;
+  }
```

Comment:
Can you add "header" (the tag) to this list? It'd be good to not have the site header on the printed version of the receipt.

Reply:
Removed. Done. 

<img width="1161" alt="Screenshot 2022-05-26 at 4 33 19 PM" src="https://user-images.githubusercontent.com/7334669/170479988-2edf8b0b-1b63-44ae-bea3-ae911cdf2de8.png">